charts/argo-cd/templates/argocd-server/service.yaml

```diff
@@ -19,14 +19,14 @@ spec:
   - name: {{ .Values.server.service.servicePortHttpName }}
     protocol: TCP
     port: {{ .Values.server.service.servicePortHttp }}
-    targetPort: {{- if .Values.server.service.namedTargetPort }} {{ .Values.server.name }} {{- else }} {{ .Values.server.containerPort }} {{- end }}
+    targetPort: {{ .Values.server.containerPorts.server }}
```

Comment:
Keeping this as a number to simplify the logic. GCE health checks require numbered ports for LoadBalancer services.